cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -258,7 +263,7 @@ configure_docker(){
 			docker_options="-D --add-runtime $runtime=$cc_runtime_bin --default-runtime=$runtime --storage-driver=overlay2"
 			modify_docker_service "$docker_options"
 		elif [ "$runtime" == "runc" ]  ; then
-			docker_options="-D"
+			docker_options="-D --storage-driver=overlay2"
```

Comment:
It might be worth creating a variable for this as it's used a couple of times (and maybe could become a cli option at some point?)

```
storage_driver="overlay2"
```